stock_picking_force_assign/tests/test_stock_picking_force_assign.py

```diff
@@ -94,7 +92,8 @@ def test_on_done_picking(self):
         self.assigned_picking.move_line_ids.qty_done = 42
         self.assigned_picking.action_done()
         self.assertEqual(self.assigned_picking.state, 'done')
-        with self.assertRaises(UserError):
+        with self.assertRaisesRegexp(
```

Comment:
In Python 3.2 and higher, assertRaisesRegex**p** is deprecated in favour of assertRaisesRegex (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex)